rest-api/src/main/resources/definitions/study_report_week.yml

```diff
@@ -0,0 +1,36 @@
+type: object
+description: One week of activities to be performed in this study by the participant, measuring weeks from the `studyStartEventId` of the study if it exists for the user (this defaults to `timeline_retrieved` for studies that do not define another event), or the first event the scheduler can find for the participant that is used by the schedule if `studyStartEventId` does not exist for the participant.
+properties:
+    weekInStudy:
+        type: integer
+        readOnly: true
+        x-nullable: false
+    startDate:
+        type: string
+        format: date
+        readOnly: true
+        x-nullable: false
```

Comment:
Missing endDate

Reply:
We never set this on the server, despite the fact there is this field on the server object. I have removed it there so it aligns with this definition of the object (and only needed to update the serialization test—truly not being used anywhere).